CONTRIBUTING.md

```diff
@@ -24,9 +24,9 @@ Code is formatted automatically and enforced by CI.
 
 Note: these instructions apply to examples configured with Bazel, see example-specific documentation for other build automation tools.
 
-Install the latest bazel version by following the steps listed <a href="https://docs.bazel.build/versions/master/install.html">here</a>
+Install the latest bazel version by following the steps listed [here](https://docs.bazel.build/versions/master/install.html).
 
-Select an example of interest from the <a href="https://github.com/open-telemetry/opentelemetry-cpp/tree/master/examples"> examples folder </a>.  Inside each example directory is a `BUILD` file containing instructions for Bazel.  Find the binary name of your example by inspecting the contents of this `BUILD` file.
+Select an example of interest from the [examples folder](https://github.com/open-telemetry/opentelemetry-cpp/tree/master/examples).  Inside each example directory is a `BUILD` file containing instructions for Bazel.  Find the binary name of your example by inspecting the contents of this `BUILD` file.
```

Comment:
It seems we are now very inconsistent with 1 space vs. 2 spaces.
Wish to get more perspectives before I try to make them consistent.

I personally vote for just having 1 space since it is easier to maintain.

Reply:
1 space looks good. Some Markdown editors (e.g. MS Word, for example) tend to insert double-space or even a space that's not 0x20 :)

Reply:
Sweet, changed all the 2 spaces to 1 space.